docs/example/tipBoxDocs.vue

```diff
@@ -25,6 +26,18 @@
             <tip-box type="definition">
                 definition
             </tip-box>
+            <tip-box type="info" icon=":rocket:">
+                info but with rocket
+            </tip-box>
+            <tip-box type="info" color="white" background-color="red">
+                info but with different colors
+            </tip-box>
+            <tip-box color="white" background-color="blue" border-color="darkblue">
+                default with custom colors
+            </tip-box>            
+            <box background-color="white" border-color="grey" border-left-color="blue">
+                empty box with left border
+            </box>
```

Comment:
- Let's move these into a separate `<div class="bs-example"></div><doc-code></doc-code>` group.
  At the moment, we need to scroll to see the doc-code.
- Two examples are sufficient: empty box (demo colours), info rocket (demo icon)
- Both can use `<box>`.
- Customise `type=default` before `type=info`.
- Be descriptive:
  - `empty box with left border` should be `default, styled as empty box with blue left border`,
  - `info but with rocket` should be `info, with rocket icon`,